lib/Doctrine/DBAL/Query/Expression/CompositeExpression.php

```diff
@@ -84,6 +88,28 @@ public function add($part)
         return $this;
     }
 
+    /**
+     * Returns a CompositeExpression with the given part added.
+     *
+     * This methods returns a new instance. The current instance is unaffected by this method call.
+     *
+     * @param self|string ...$parts
+     */
+    public function with(...$parts) : self
```

Comment:
The signature should require at least one argument.

Reply:
Fixed in ea51a40e342861a739108311298f8d8ccdf38ed3 and 2b01c576ebe2206683bceaad0c7d5f26f77850fd.